extra/intl-extra/tests/IntlExtensionTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Extra\Intl\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Twig\Extra\Intl\IntlExtension;
+
+class IntlExtensionTest extends TestCase
+{
+    public function testConstructor()
```

Comment:
I would rather name this test `testFormatterProto`, as it is about testing the case where you use a custom proto in the constructor, not about testing all cases of the constructor (which would be a bad idea for a single test anyway)

Reply:
done